sentry-core/src/main/java/io/sentry/core/SentryEvent.java

```diff
@@ -19,7 +19,7 @@
   private String dist;
   private String logger;
   private SentryValues<SentryThread> threads;
-  private SentryValues<SentryException> exceptions;
+  private SentryValues<SentryException> exception;
```

Comment:
`SentryValues` is a list.
```suggestion
  private SentryValues<SentryException> exceptions;
```

Reply:
it's called `exception` though.